api/app/assets/fonts/License

```diff
@@ -0,0 +1,202 @@
+

```

Comment:
I think this file does not belong in the `assets/fonts` directory. 
Probably it would be best to have a license disclaimer also for frontend/backend libs (You can generate with `yarn licenses generate-disclaimer`).

Reply:
But is scope of different PR